{"uuid": "a38fe640-fd6b-4d5f-b152-f432784f8de7", "children": ["6573dd7a-2874-4c13-96e6-249490924b9c", "718e67ed-63e4-4e09-b85d-6ada74540008", "cceaf6b3-aa05-4ca5-ab51-e910898f527b", "ab68571c-2a85-4b52-b25e-cc5c842f8199"], "befores": [{"name": "get_token", "status": "broken", "statusDetails": {"message": "AttributeError: 'Logger' object has no attribute 'logger'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/tim/Desktop/test_pro/pyframe/testcases/conftest.py\", line 32, in get_token\n    log.logger.info(\"Authorization:Bearer \"+token[0])\n    ^^^^^^^^^^\n"}, "start": 1716100267756, "stop": 1716100268072}], "start": 1716100267756, "stop": 1716100268160}